src/main/kotlin/com/boostyboys/mcs/Application.kt

```diff
@@ -37,10 +40,21 @@ private const val LOCAL_PORT = 8080
 /**
  * An entry point of the embedded-server
  **/
+@OptIn(ExperimentalSerializationApi::class)
 fun main() {
     val port = System.getenv("PORT")?.toInt() ?: LOCAL_PORT
     embeddedServer(Netty, port = port) {
         kodeinApplication {
+            install(ContentNegotiation) {
```

Comment:
we already have this installed in the bindHttpClient method - is this doing something different?

Reply:
same as the other comment